docs/openapi/openapi.yaml

```diff
@@ -219,16 +219,69 @@ components:
     InitialLpa:
       type: object
       required:
+        - lpaType
         - donor
         - attorneys
+        - certificateProvider
+        - signedAt
       properties:
+        lpaType:
+          type: string
+          enum:
+            - property-and-affairs
+            - personal-welfare
         donor:
           $ref: "#/components/schemas/Donor"
         attorneys:
           type: array
           items:
             $ref: "#/components/schemas/Attorney"
           minLength: 1
+        certificateProvider:
+          $ref: "#/components/schemas/CertificateProvider"
+        peopleToNotify:
+          type: array
+          items:
+            $ref: "#/components/schemas/PersonToNotify"
+        howAttorneysMakeDecisions:
+          type: string
+          enum:
+            - jointly
+            - jointly-and-severally
+            - jointly-for-some-severally-for-others
+        howAttorneysMakeDecisionsDetails:
+          type: string
+        howReplacementAttorneysMakeDecisions:
+          type: string
+          enum:
```

Comment:
Is it going to be possible to have this schema refer to a glossary of terms, which also has versioning and history of those terms? That way we would know what "jointly" meant at the point it was used in an LPA. Same goes for other choices the donor might make on the LPA.